packages/cli/src/validateNonInterActiveAuth.test.ts

```diff
@@ -175,4 +212,29 @@ describe('validateNonInterActiveAuth', () => {
     expect(consoleErrorSpy).toHaveBeenCalledWith('Auth error!');
     expect(processExitSpy).toHaveBeenCalledWith(1);
   });
+
+  it('skips validation if useExternalAuth is true', async () => {
+    // Mock validateAuthMethod to return error to ensure it's not being called
+    const mod = await import('./config/auth.js');
```

Comment:
nit: a bit silly we are importing inline like this. unless that is strictly needed move that to a regular top level import.